### PR TITLE
Update email template docs

### PR DESCRIPTION
**Pull request by lyyder**
- Update context description in email templates reference article
- Add an explanation on accessing email context properties to the Edit templates with CLI tutorial

### CODE REVIEW

File: src/docs/references/03-email-templates/index.md [hunk 1]
```diff
@@ -331,22 +331,28 @@ Context for transaction emails:
...
+    "private-data": "extended-data",
+    "public-data": "extended-data",
+    "protected-data": "extended-data"
   },
   "marketplace": {
     "name": "string",
-    "url" "string"
+    "url": "string"
   },
   "recipient-role": {"one-of": ["provider", "customer"]},
```
Comment (kpuputti) [inline src/docs/references/03-email-templates/index.md]: Could you change these `one-of` things? I think the syntax is really weird and could be a lot simpler:

```suggestion
   "recipient-role": "provider", // either "provider" or "customer"
```

File: src/docs/references/03-email-templates/index.md [hunk 2]
```diff
@@ -331,22 +331,28 @@ Context for transaction emails:
...
-    "display-name": "string"
+    "display-name": "string",
+    "private-data": "extended-data",
+    "public-data": "extended-data",
+    "protected-data": "extended-data"
   },
   "transaction": {
     "id": "uuid",
-    "line-items": [
+    "tx-line-items": [
```
Comment (kpuputti) [inline src/docs/references/03-email-templates/index.md]: Has this changed or was the reference wrong?

Comment (lyyder) [reply src/docs/references/03-email-templates/index.md]: @kpuputti It has changed.

File: src/docs/references/03-email-templates/index.md (hunk 1 above)
Comment (lyyder) [reply src/docs/references/03-email-templates/index.md]: @kpuputti or what about
```
"recipient-role": "string", // either "provider" or "customer"
```
?

Comment (lyyder) [reply src/docs/references/03-email-templates/index.md]: Changing to the above to get this forward.

File: src/docs/references/03-email-templates/index.md (hunk 2 above)
Comment (kpuputti) [reply src/docs/references/03-email-templates/index.md]: Ok, good to know, just made sure. I assume all the deprecation, possible braking changes and other stuff is taken into account.

Comment (lyyder) [reply src/docs/references/03-email-templates/index.md]: Yep, so `line-items` is still available in the context but not documented anywhere.

**Pull request merged**